src/formatDate.js

```diff
@@ -8,7 +8,54 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const [oldData, idFromYear, oldYear, fromSumbol] = getFormat(fromFormat);
+  const [newData, , newYear, toSumbol] = getFormat(toFormat);
```

Comment:
The variable `toSumbol` is misspelled. It should be `toSymbol` to correctly represent the delimiter symbol in the date format.